src/coreclr/src/jit/lclvars.cpp

```diff
@@ -5131,10 +5131,26 @@ bool Compiler::lvaIsPreSpilled(unsigned lclNum, regMaskTP preSpillMask)
 }
 #endif // TARGET_ARM
 
-/*****************************************************************************
- *  lvaUpdateArgsWithInitialReg() : For each argument variable descriptor, update
- *  its current register with the initial register as assigned by LSRA.
- */
+//------------------------------------------------------------------------
+// UpdateLifeFieldVar: Update live sets for only the given field of a multi-reg LclVar node.
+//
+// Arguments:
+//    lclNode - the GT_LCL_VAR node.
```

Comment:
Nit: it looks like the header was not updated for this function.

Reply:
Thanks for catching that; rather than fix in this PR, I've fixed it in the follow-up PR #39326 which I'll rebase on this once this one's merged.